src/Files.App/Data/Models/SelectedItemsPropertiesViewModel.cs

```diff
@@ -165,6 +172,13 @@ public bool ItemSizeVisibility
 			set => SetProperty(ref itemSizeVisibility, value);
 		}
 
+		private bool itemSizeOnDiskVisibility = false;
+		public bool ItemSizeOnDiskVisibility
```

Comment:
I think we can avoid this by creating a converter that's false when the string is empty.

Reply:
Ok, I will change this